test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Index.Encoded.html

```diff
@@ -1,10 +1,10 @@
 <html>
 <body>
     <div>
-        <a title="&lt;the title>" href="HtmlEncode[[/HtmlGeneration_Product]]">Product Index</a>
+        <a title="&lt;the title>" href="HtmlEncode[[/Product/Index]]">Product Index</a>
```

Comment:
Look at this for understanding why I am making this change: https://github.com/aspnet/Mvc/pull/8143/files#diff-4b5a6a9e93ff25e0df39bf957e637b1f
Basically there is no `ProductController`, but there is a `HtmlGeneration_ProductController`